spec/models/user_spec.rb

```diff
@@ -0,0 +1,72 @@
+require "rails_helper"
+RSpec.describe User, type: :model do
+  let!(:user) {FactoryBot.build :user}
+  let!(:user_fail) {FactoryBot.build :user, email: nil}
+
+  describe "Validations" do
+    it "required all field" do
```

Comment:
describe, context, it là gì ?

Reply:
it: dùng để mô tả test case 
describe: dùng để mô tả một nhóm các test case theo class, method (action trong controller)
context: để nhóm theo điều kiện của nhóm testcacse